### PR TITLE
Add requester pays samples

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

**Comment by ace-n** [comment]
Done - FYI, I wasn't sure whether or not this sample warranted a `README` update since `requesterPays.js` struck me as a mix of stuff from other files.

File: storage/requesterPays.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+  // Gets the requester-pays status of a bucket
+  storage
+    .bucket(bucketName)
+    .getMetadata()
+    .then((data) => {
+      let status;
+      const metadata = data[0];
+      if (metadata && metadata.billing && metadata.billing.requesterPays) {
+        status = `enabled`;
```
Comment (frankyn) [inline storage/requesterPays.js]: Should this string use single-quotes?

File: storage/README.md [hunk 2]
```diff
@@ -199,6 +200,39 @@ For more information, see https://cloud.google.com/storage/transfer
...
+                                                      bucket.
+  download <bucketName> <srcFileName> <destFileName>  Downloads a file from a bucket using requester-pays requests.
+
+Options:
+  --help  Show help                                                                                            [boolean]
+
+Examples:
+  node requesterPays.js enable my-bucket                        Enables requester-pays requests on a bucket named
+                                                                "my-bucket".
+  node requesterPays.js list                                    Disables requester-pays requests on a bucket named
```
Comment (ace-n) [inline storage/README.md]: Fixed 😄  - good catch!

File: storage/requesterPays.js (hunk 1 above)
Comment (ace-n) [inline storage/requesterPays.js]: Technically, the other strings should use backticks. Will fix.

File: storage/requesterPays.js [hunk 3]
```diff
@@ -177,8 +177,8 @@ const cli = require(`yargs`)
     (opts) => downloadFileUsingRequesterPays(opts.projectId, opts.bucketName, opts.srcFileName, opts.destFileName)
   )
   .example(`node $0 enable my-bucket`, `Enables requester-pays requests on a bucket named "my-bucket".`)
-  .example(`node $0 list`, `Disables requester-pays requests on a bucket named "my-bucket".`)
-  .example(`node $0 delete my-bucket`, `Determines whether requester-pays requests are enabled for a bucket named "my-bucket".`)
+  .example(`node $0 disable`, `Disables requester-pays requests on a bucket named "my-bucket".`)
```
Comment (frankyn) [inline storage/requesterPays.js]: One last nit, disable should accept a bucket id similar to enable.

**Pull request merged**